index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <a href="">My Logo</a>
+        <nav> <img src="images/Union.png" alt="hamburgeur menu"></nav>
+    </header>
+    <main>
+        <section>
+          
+            <h1>
+                I’m Pamphile <br>
+                Glad to see <br>
+                   you!
+            </h1>
+            
+          <p>
+            I’m a software developer!
+             I can help you build a product ,
+              feature or website Look through some of my work and experience!
+               If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+          </p>
+    
+          <div class="social-media">
+    
+          <p class="social-text">LET’S CONNECT</p>
```

Comment:
- [ ] Still, your font family is not working so Kindly import the font family and use font-family: 'Poppins'.